website/content/docs/connect/gateways/api-gateway/secure-traffic/encrypt-vms.mdx

```diff
@@ -30,12 +30,12 @@ with Consul API gateway configurations.
 
 ## Define TLS certificates
 
-1. Create an [`inline-certificate` configuration entry](/consul/docs/connect/gateways/api-gateway/configuration/inline-certificate) and specify the following fields:
-   - `Kind`: Specifies the type of configuration entry. This must be set to `inline-certificate`.
+1. Create a [file system](/consul/docs/connect/config-entries/file-system-certificate) or [inline certificates](/consul/docs/connect/config-entries/inline-certificate) and specify the following fields:
+   - `Kind`: Specifies the type of configuration entry. This must be set to `file-system-certificate` or `inline-certificate`.
    - `Name`: Specify the name in the [API gateway listener configuration](/consul/docs/connect/gateways/api-gateway/configuration/api-gateway#listeners) to bind the certificate to that listener.  
-   - `Certificate`: Specifies the inline public certificate to use for TLS as plain text.
-   - `PrivateKey`: Specifies the inline private key to use for TLS as plain text.
-1. Configure any additional fields necessary for your use case, such as the namespace or admin partition. Refer to the [`inline-certificate` configuration entry](/consul/docs/connect/gateways/api-gateway/configuration/inline-certificate) reference for additional information.
+   - `Certificate`: Specifies the filepath to the certificate on the local system or the inline public certificate as plain text.
+   - `PrivateKey`: Specifies the filepath to private key on the local system or the inline private key to  as plain text.
+1. Configure any additional fields necessary for your use case, such as the namespace or admin partition. Refer to the [`file-system-certificate` configuration entry reference](/consul/docs/connect/config-entries/file-system-certificate) or [`inline-certificate` configuration entry reference](/consul/docs/connect/gateways/api-gateway/configuration/inline-certificate) for more information.
 1. Save the configuration.
 
 The following example defines a certificate named `my-certificate`. API gateway configurations that specify `inline-certificate` in the `Certificate.Kind` field and `my-certificate` in the `Certificate.Name` field are able to use the certificate.
```

Comment:
The example is only related to inline certificate We should introduce tabs or, since FS certificate is the preferred one, if we only want to show a single example we should use an FS certificate.